tests/e2e_test/module_deletion_test.go

```diff
@@ -104,7 +104,7 @@ var _ = Describe("Non Blocking Kyma Module Deletion", Ordered, func() {
 				WithArguments(runtimeClient, defaultRemoteKymaName, remoteNamespace, module).
 				Should(Succeed())
 			By("Then the Module will be updated on SKR")
-			Eventually(DeploymentIsReady).
+			Consistently(DeploymentIsReady).
```

Comment:
I believe this step will fail, it's a new deploy, can't be consistently in ready

Reply:
You are correct, that was a stupid change.